src/Common/Query/ReverseQuery.php

```diff
@@ -64,7 +64,7 @@ public static function create(Coordinates $coordinates)
      *
      * @return ReverseQuery
      */
-    public static function fromCoordinates($latitude, $longitude): self
+    public static function fromCoordinates(float $latitude, float $longitude): self
     {
```

Comment:
Could this break existing code where an int was passed?

Reply:
See https://github.com/geocoder-php/Geocoder/pull/1073#issuecomment-653751763

Reply:
Oops sorry didn't see that comment 🙈